static/micro-blog.css

```diff
@@ -1,3 +1,3 @@
base-topbar-container .lb24-base-topbar-status .lb24-base-topbar-status-updating{color:#999}#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-status .lb24-base-topbar-status-text{display:inline-block;line-height:25px;margin-top:20px;color:#999}#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-playback,#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-sort,#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-volume{display:inline-block;float:right;width:25px;height:25px;text-align:center;cursor:pointer}#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-playback>div,#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-sort>div,#LB24 .lb24-theme-card .lb24-base-topbar-container .lb24-base-topbar-volume>div{display:inline-block;line-height:25px;margin-top:20px;color:grey}#LB24 .lb24-theme-card .lb24-base-score-container{background:#fff;border-top:1px solid #bc3832;padding:20px 15px 15px;margin:30px 0}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info{display:-ms-flexbox;display:flex;width:100%}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info>div{padding:0 8px;-ms-flex-item-align:center;align-self:center}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-name,#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-nameshort{-ms-flex:1;flex:1;color:#141414}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-name.lb24-base-score-a,#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-nameshort.lb24-base-score-a{text-align:right}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-name.lb24-base-score-b,#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-nameshort.lb24-base-score-b{text-align:left}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-nameshort{display:none}@media(max-width:580px){#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-nameshort{display:block}}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-name{display:none}@media(min-width:581px){#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-name{display:block}}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-logo>div{width:36px;height:36px;background-repeat:no-repeat;background-position:0 0;background-size:100% auto}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-num{font-size:18px;width:30px;color:#141414;text-align:center}#LB24 .lb24-theme-card .lb24-base-score-container .lb24-base-score-info .lb24-base-score-vs>.icon-vs{font-size:23px;color:#bc3832}#LB24 .lb24-theme-card .lb24-base-score-contai
```

Comment:
@mishs remember I said we can't just copy their CSS because it's their copyright. Please remove their CSS and style it so it the markup we're hardcoding looks like it used to but with our own CSS

Reply:
Commits below in response:
260bed2
da33cfd
04e4828
-Removes copyright images and links
-converts styling to our own css